penlight-dev-1.rockspec

```diff
@@ -1,18 +1,16 @@
-local package_name = "penlight"
-local package_version = "dev"
-local rockspec_revision = "1"
-local github_account_name = "lunarmodules"
-local github_repo_name = package_name
-local git_checkout = package_version == "dev" and "master" or package_version
-
+package = "penlight"
```

Comment:
I prefer the vertical alignment of the "local"s, aesthetically. Despite this is just copying one variable into another.

(hence also the 2 lines white-space below to accentuate the vertical alignment above)